tests/stacking.rs

```diff
@@ -15,6 +15,13 @@ fn concatenating() {
     let d = concatenate![Axis(0), a.row(0), &[9., 9.]];
     assert_eq!(d, aview1(&[2., 2., 9., 9.]));
 
+    let d = concatenate![Axis(1), a.row(0).insert_axis(Axis(1)), aview1(&[9., 9.]).insert_axis(Axis(1))];
+    assert_eq!(d, aview2(&[[2., 9.],
+                           [2., 9.]]));
+
+    let d = concatenate![Axis(0), a.row(0).insert_axis(Axis(1)), aview1(&[9., 9.]).insert_axis(Axis(1))];
+    assert_eq!(d.t(), aview2(&[[2., 2., 9., 9.]]));
```

Comment:
It took me a while to understand this because I didn't notice the `.t()` at first. This is easier for me to read:

```suggestion
    assert_eq!(d, aview2(&[[2.], [2.], [9.], [9.]]));
```